amr-wind/ocean_waves/relaxation_zones/stokes_waves_ops.H

```diff
@@ -101,8 +101,8 @@ struct UpdateRelaxZonesOp<StokesWaves>
                         amrex::Real eta{0.0}, u_w{0.0}, v_w{0.0}, w_w{0.0};
 
                         relaxation_zones::stokes_waves(
-                            order, wavelength, waterdepth, waveheight, x, time,
-                            eta, u_w, v_w, w_w);
+                            order, wavelength, waterdepth, waveheight, x, z,
```

Comment:
I think z should be redefined as (problo[2] + (k + 0.5) * dx[2] - zsl) in line 99. In case of nonzero zsl. This should also be done for linear waves.